projects/emissary-ingress/emissary/helm/patches/0001-helm-changes.patch

```diff
@@ -83,6 +100,56 @@ index b974e5198..bab4360f7 100644
 +- https://github.com/datawire/ambassador
 +- https://github.com/prometheus/statsd_exporter
 +version: 3.0.0-cbf68a7dd53dda13c681af3e19660a5605050839
+diff --git a/charts/emissary-ingress/ci.in/tests/manifests/backend.yaml b/charts/emissary-ingress/ci.in/tests/manifests/backend.yaml
+index b2d9205df..2a23de707 100644
+--- a/charts/emissary-ingress/ci.in/tests/manifests/backend.yaml
++++ b/charts/emissary-ingress/ci.in/tests/manifests/backend.yaml
+@@ -2,38 +2,38 @@
+ apiVersion: getambassador.io/v1
+ kind: Mapping
+ metadata:
+-  name: quote-backend
```

Comment:
How does this work?

Reply:
This is part of the tests, but can be removed for now anyway.